src/calculateRentalCost.js

```diff
@@ -20,7 +20,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const discountFor3Days = 20;
+  const discoutnFor7Days = 50;
+  const carRentalPrice = 40;
+  if (days >= 3 && days < 7) {
+    return days * carRentalPrice - discountFor3Days;
+  } else if (days >= 7) {
+    return days * carRentalPrice - discoutnFor7Days;
+  } return days * carRentalPrice;
```

Comment:
Итоговый return должен быть на новой строке и отделен отступом
```suggestion
  } 

return days * carRentalPrice;
```